smart-contracts/scripts/lock/clone.js

```diff
@@ -11,7 +11,6 @@ async function main({
   const PublicLock = await ethers.getContractFactory('PublicLock')
   const lock = PublicLock.attach(lockAddress)
 
-  // eslint-disable-next-line no-console
```

Comment:
I like the `no-console` rule has it enforced we keep code clean. wdyt?

Reply:
I don't think it will add value. We are using console in every package and app. Someone will end up disabling the rule to add a console log like they have everywhere already. 

Console log is relatively harmless so I think if we want it somewhere, explicitly enable it in eslint rules for that project.

Reply:
the idea behind making the use of console log explicit is to make sure it was intended, so we dont forget any dev-related logs that could potentially leak credentials - and keep tests clean as they sometimes are reviewed less thoroughly so logs tends to remain there. 

Anyways, not a dealbreaker just some thoughts

Reply:
I will add a warning then (no error).